src/drivers/uavcan/libdronecan/libuavcan/include/uavcan/protocol/node_status_monitor.hpp

```diff
@@ -10,6 +10,8 @@
 #include <uavcan/node/subscriber.hpp>
 #include <uavcan/node/timer.hpp>
 #include <uavcan/protocol/NodeStatus.hpp>
+#include <uORB/Publication.hpp>
+#include <uORB/topics/dronecan_node_status.h>
```

Comment:
We probably don't want to couple uORB into the dronecan library. Can you make an interface instead and add the uORB plumbing to **uavcan_main.cpp/hpp**

Reply:
Alrighty, I've taken all uorb references out of the dronecan library, and all publishing has been offloaded to uavcan main